nixos/modules/services/networking/quorum.nix

```diff
@@ -0,0 +1,202 @@
+{ config, pkgs, lib, ... }:
+let
+
+  inherit (lib) mkEnableOption mkIf mkOption literalExample types optionalString;
+
+  cfg = config.services.quorum;
+  genesisFile = pkgs.writeText "genesis.json" (builtins.toJSON cfg.genesis);
+  staticNodesFile = pkgs.writeText "static-nodes.json" (builtins.toJSON cfg.staticNodes);
+
+in {
+  options = {
+
+    services.quorum = {
+      enable = mkEnableOption "Bitcoin daemon";
+
+      dataDir = mkOption {
+        type = types.path;
+        default = "/var/lib/quorum";
+        description = "The data directory for quorum.";
+      };
```

Comment:
I was advised to not have `dataDir` configurable unless it's usual to changet, maybe it's also the case here?

Reply:
I thought about the use case when someone wants to have that directory elsewhere (like ```/opt```) but I just realized, this is not possible with systemd.

Reply:
Can you bind-mount it perhaps?

Reply:
I will look into it tomorrow, but probably will leave it as up to the end user to bind mount if needed.

Reply:
No need, was just curious if there's a solution for users that want dataDir somewhere else.

Reply:
I will just remove it then.